features/luks/data/lib/uevent/handlers/085-luks

```diff
@@ -48,8 +48,26 @@ match_dev_in_array() {
 	return 1
 }
 
+luks_unl0kr_ask_pass() {
+	[ "$#" = 2 ] || return 1
+
+	local cmd="$1" tries=$2 rc=2
```

Comment:
I'd prefer the following:
```shell
local cmd tries rc

rc=2
cmd="$1"; shift
tries="$1"; shift
```
But it's up to @legionus

Reply:
Yes, I like @wladmis's version better. All variables are quoted.